File: api/src/main/java/com/sniper/music/api/DefaultOkHttpConfig.java
package com.sniper.music.api;

public class DefaultOkHttpConfig implements OKHttpConfig {

    private static final long CONNECTION_TIMEOUT = 30;
    private static final long READ_TIMEOUT = 30;
    private static final long WRITE_TIMEOUT = 30;

    @Override
    public long getConnectTimeout() {
        return CONNECTION_TIMEOUT;
    }

    @Override
    public long getReadTimeout() {
        return READ_TIMEOUT;
    }

    @Override
    public long getWriteTimeout() {
        return WRITE_TIMEOUT;
    }
}

File: api/src/main/java/com/sniper/music/api/okhttp/DefaultOkHttpConfig.kt
package com.sniper.music.api.okhttp

class DefaultOkHttpConfig : OKHttpConfig {

    override val connectTimeout: Long
        get() = CONNECTION_TIMEOUT

    override val readTimeout: Long
        get() = READ_TIMEOUT

    override val writeTimeout: Long
        get() = WRITE_TIMEOUT

    companion object {

        private const val CONNECTION_TIMEOUT: Long = 30
        private const val READ_TIMEOUT: Long = 30
        private const val WRITE_TIMEOUT: Long = 30
    }
}
